iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -54,8 +56,9 @@ protected AbstractTapChanger(Ref<? extends VariantManagerHolder> network, H pare
         for (int i = 0; i < variantArraySize; i++) {
             this.tapPosition.add(tapPosition);
             this.regulating.add(regulating);
-            this.targetDeadband.add(targetDeadband);
+            this.targetDeadband.add(Double.isNaN(targetDeadband) ? 0 : targetDeadband);
```

Comment:
Could probably be cached in a local variable.

Looking to the code below, it seems there is a `checkDeadband` method that throws when value is NaN